web/vtadmin/tailwind.config.js

```diff
@@ -48,9 +48,8 @@ module.exports = {
             },
         },
         fontFamily: {
-            mono: ['NotoMono', 'source-code-pro', 'menlo', 'monaco', 'consolas', 'Courier New', 'monospace'],
+            mono: ['source-code-pro', 'menlo', 'monaco', 'consolas', 'Courier New', 'monospace'],
```

Comment:
This change is just for @planetscale/vtadmin and is important to making asset resolution in our Rails app easier. This likely won't run into any merge conflicts with OS VTAdmin.